app/main.py

```diff
@@ -1,3 +1,127 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+import os.path
+
+from app.customer import Customer
+from app.shop import Shop
+from app.car import Car
+
+
+def price_of_gasoline(
```

Comment:
Function name `price_of_gasoline` is not descriptive enough. Consider renaming it to `calculate_gasoline_cost` or similar to better reflect its purpose.